README.md

```diff
@@ -10,17 +10,18 @@ Also, see the [API documentation](https://aws.github.io/aws-encryption-sdk-c/htm
 ## Dependencies
 
 The only direct dependencies of this code are OpenSSL 1.0.2 or higher or 1.1.0 or higher and
-[aws-c-common](https://github.com/awslabs/aws-c-common) v0.3.0 or higher. You will also need
+[aws-c-common](https://github.com/awslabs/aws-c-common) v0.3.15. You will also need
```

Comment:
Why aren't we updating aws-c-commons to the latest version? https://github.com/awslabs/aws-c-common/releases. The current release is v0.4.15

Reply:
Version bump is pending in [aws-sdk-cpp PR 1267](https://github.com/aws/aws-sdk-cpp/pull/1267).